File: app/src/main/java/acr/browser/lightning/search/engine/StartPageMobileSearch.java
package acr.browser.lightning.search.engine;

import acr.browser.lightning.R;
import acr.browser.lightning.constant.Constants;

/**
 * The StartPage mobile search engine.
 */
public class StartPageMobileSearch extends BaseSearchEngine {

    public StartPageMobileSearch() {
        super("file:///android_asset/startpage.png", Constants.STARTPAGE_MOBILE_SEARCH, R.string.search_engine_startpage_mobile);
    }

}

File: app/src/main/java/acr/browser/lightning/search/engine/StartPageMobileSearch.kt
package acr.browser.lightning.search.engine

import acr.browser.lightning.R

/**
 * The StartPage mobile search engine.
 */
class StartPageMobileSearch : BaseSearchEngine(
        "file:///android_asset/startpage.png",
        "https://startpage.com/do/m/mobilesearch?language=english&query=",
        R.string.search_engine_startpage_mobile
)
